install/pre-install-payload/scripts/reqs-deploy.sh

```diff
@@ -178,60 +178,86 @@ function configure_nydus_snapshotter_for_containerd() {
 	echo "configure nydus snapshotter for containerd"
 
 	local containerd_imports_path="/etc/containerd/config.toml.d"
+	local tmp_containerd_config="$(mktemp)"
 
-	echo "Create ${containerd_imports_path}"
-	mkdir -p "${containerd_imports_path}"
+	( for i in {{1..10}}; do
```

Comment:
One small && future improvement (as we want to merge this asap): print a message on begin of each iteration number for debugging sake.

Reply:
I though about it but decided not to pollute the log as failed iteration won't cause anything (tmp files are re-used and the service is unaffected) and if we go over all attempts we'll get a message. Anyway if you insist I can add it...

Reply:
That's okay, if we ever need to print a message for debug in the future then we can implement.